sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderer.kt

```diff
@@ -60,14 +60,48 @@ internal abstract class MapboxRenderer : MapClient {
 
   @AnyThread
   override fun scheduleRepaint() {
-    renderThread.requestRender()
+    Logger.e("KIRYLDD", "scheduleRepaint!")
+    renderThread.queueRenderEvent(
+      RenderEvent(
```

Comment:
Maybe create single object instance `val repaintRenderEvent = RenderEvent()` since we don't need distinct instances?

Reply:
there could be multiple instances obviously coming from any places and any threads. In that case it's rather logical to create some pool but I don't want to overcomplicate it

Reply:
However indeed, in this particular case we can create it once. Thanks for catching!